Feature/ds 252 - Updated documentation voice and tone, as well as added some uniformity. (#383)
gsoft-inc/wl-hopper: apps/docs/content/components/concepts/responsive-styles.mdx

---
title: Responsive Styles
description: Hopper style props accept a specialized syntax to support responsive breakpoints. These responsive properties help build adaptive user interfaces.
order: 3
---

## Introduction

In addition to static values, all style props support object syntax to specify different values for the prop depending on a responsive breakpoint. Breakpoints are named following t-shirt sizing, and correspond to common device resolutions.

### Example

A `Div` with a default background color overridden at each breakpoint. Resize your browser window to see this in action.

<Example src="HopperProvider/docs/responsive-styles/introduction" isOpen />

## Breakpoints

> There are five breakpoints available (in addition to the base), inspired by typical device resolutions.

Reviewer: ```suggestion
In addition to the base, there are five breakpoints inspired by typical device resolutions.
```

<BreakpointTable />

## Mobile-first

> Hopper uses a mobile-first breakpoint system, similar to what you might be used to in other frameworks such as Bootstrap and Tailwind.

Reviewer: ```suggestion
Hopper uses a mobile-first breakpoint system, similar to those in frameworks like Bootstrap and Tailwind.
```

<Example src="HopperProvider/docs/responsive-styles/mobile-first-do" isOpen />

> It's often a good idea to implement the mobile layout for a design first, then gradually add styles as the screen size increases and the design requires, progressing from small screens to wider screens.

Reviewer: This sentence seems a bit weird to me.. specifically the part that says "and the design requires". Maybe we could simplify to something like this:
```suggestion
It’s often best to start with the mobile first layout for a design, then add styles as the screen size increases, moving from small to larger screens.
```
